cfn-resources/alert-configuration/template.yml

```diff
@@ -21,7 +21,9 @@ Resources:
       Handler: handler
       Runtime: go1.x
       CodeUri: bin/
-      Environment: 
-        Variables: 
+      Environment:
+        Variables:
           MODE: Test
+          LOG_LEVEL: debug
+          MONGODB_ATLAS_OPS_MANAGER_URL:
```

Comment:
[q, not blocking] Are these .tpl generated from a script or the `cfn` command line? I'm conscious that we will need to add these variables manually for new resources.

Can we improve the process here?

Reply:
> Are these .tpl generated from a script or the cfn command line? 
These template files are generated when running `cfn init`

Reply:
>  I'm conscious that we will need to add these variables manually for new resources.
Can we improve the process here?

Need to see if there is a script that has been used to generate a cfn resource instead of just running `cfn init`, if not we can create this script to add automatically these env var to a template. Maybe even running this script as part of a git hook.

Reply:
There is a wrapper used today to generate the resources - could be a good starting point. Again, not needed right now, improvement ticket idea

https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/aebdeec84318a76b116d4eaf6a3ce22d0a76fc8b/cfn-resources/autogen/Makefile#L22